lib/zlib.js

```diff
@@ -453,7 +453,9 @@ Zlib.prototype.flush = function(kind, callback) {
   }
 };
 
-Zlib.prototype.close = function(callback) {
+Zlib.prototype.close = function(callback, options) {
+  options = options || {};
```

Comment:
`Object.assign()` is safer than this pattern in the event that a non-object truthy value is passed. Not sure if we want to cover up those types of programming errors, but thought I'd at least mention it.